CRM/Member/BAO/MembershipType.php

```diff
@@ -832,6 +833,12 @@ public static function getAllMembershipTypes() {
         if (isset($type['contribution_type_id'])) {
           unset($types[$id]['contribution_type_id']);
         }
+        $types[$id]['tax_rate'] = (float) ($taxRates[$type['financial_type_id']] ?? 0.0);
+        $multiplier = 1;
+        if ($types[$id]['tax_git rate'] !== 0.0) {
```

Comment:
@eileenmcnaughton Is the git tax based on the number of lines of code added, or removed?

Reply:
argh - fixed